.editorconfig

```
@@ -0,0 +1,18 @@
+root = true
+
+[*.{js,ts,tsx}]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+max_line_length = 100
```

```suggestion
max_line_length = 80
```
저희 프리티어 설정에 맞춰야 할 것 같습니다!

---

[수정완료 ! ](https://github.com/price-offer/offer-fe/pull/86/commits/ece8c0b752cff305299353354edf034feab2ba2f)

---

효정넴 링크가 깨진 것 같아요!! 확인은 했습니다! 감사합니다!